lib/rubocop/comment_config.rb

```diff
@@ -20,6 +20,12 @@ class CommentConfig
         .gsub(' ', '\s*')
     )
 
+    # @api private
+    COMMENT_NEXT_LINE_DIRECTIVE_REGEXP = Regexp.new(
+      "# rubocop : ((?:disable-next-line))\\b #{COPS_PATTERN}"
+        .gsub(' ', '\s*')
```

Comment:
I didn't realize that we supported so much white space in the comments. I see we're doing the same thing above so this looks good to me.